cognite/extractorutils/logging_prometheus.py

```diff
@@ -0,0 +1,29 @@
+import logging
+
+from prometheus_client import Counter
+
+log_entries = Counter(
+    "cognite_python_logging_messages_total", "Count of log entries by logger and level.", ["logger", "level"]
+)
+
+
+class ExportingLogHandler(logging.Handler):
+    """A LogHandler that exports logging metrics for Prometheus.io."""
+
+    def emit(self, record):
+        log_entries.labels(record.name, record.levelname).inc()
+
+
+def export_log_stats_on_root_logger(logger=logging.getLogger()):
```

Comment:
@pavelzubarev will this get the correct logger?

Reply:
yes. that will get the root logger by default